packages/compiler/src/template_parser/binding_parser.ts

```diff
@@ -244,6 +244,9 @@ export class BindingParser {
       targetProps: ParsedProperty[], keySpan?: ParseSourceSpan) {
     if (isAnimationLabel(name)) {
       name = name.substring(1);
+      keySpan = keySpan &&
+          moveParseSourceSpan(
+                    keySpan, new AbsoluteSourceSpan(keySpan.start.offset + 1, keySpan.end.offset));
```

Comment:
style: The code style in the compiler is to use explicit equality checks:
```suggestion
      if (keySpan !== undefined) {
        keySpan = moveParseSourceSpan(
                    keySpan, new AbsoluteSourceSpan(keySpan.start.offset + 1, keySpan.end.offset));
      }
```
Please update this here and in the other changes